### PR TITLE
Node-1186 Test for IssueTransactionV2

### PR DESCRIPTION
**Pull request by monroid**

### CODE REVIEW

File: it/src/test/scala/com/wavesplatform/it/sync/transactions/IssueTransactionSuite.scala [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import com.wavesplatform.it.api.SyncHttpApi._
+import com.wavesplatform.it.transactions.BaseTransactionSuite
+import com.wavesplatform.it.util._
+import com.wavesplatform.it.sync._
+import com.wavesplatform.transaction.smart.script.ScriptCompiler
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class IssueTransactionSuite extends BaseTransactionSuite with TableDrivenPropertyChecks {
+  val script            = ScriptCompiler(s"""true""".stripMargin).explicitGet()._1.bytes.value.base64
+  val supportedVersions = List(null) // add "2" after IssueTransactionV2 activation
```
Comment (ismagin) [inline it/src/test/scala/com/wavesplatform/it/sync/transactions/IssueTransactionSuite.scala]: Plese find a better way than `null`

File: it/src/test/scala/com/wavesplatform/it/sync/transactions/IssueTransactionSuite.scala [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  test("asset issue changes issuer's asset balance; issuer's waves balance is decreased by fee") {
+    for (v <- supportedVersions) {
+      val scriptText       = if (Option(v).isDefined) script else null
```
Comment (ismagin) [inline it/src/test/scala/com/wavesplatform/it/sync/transactions/IssueTransactionSuite.scala]: null

File: it/src/test/scala/com/wavesplatform/it/sync/transactions/IssueTransactionSuite.scala [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+  test("Able to create asset with the same name") {
+    for (v <- supportedVersions) {
+      val scriptText       = if (Option(v).isDefined) script else null
+      val assetName        = "myasset1"
+      val assetDescription = "my asset description 1"
+      val (balance1, eff1) = notMiner.accountBalances(firstAddress)
+
+      val issuedAssetId =
+        sender.issue(firstAddress, assetName, assetDescription, someAssetAmount, 2, reissuable = false, issueFee, script = Some(scriptText)).id
```
Comment (ismagin) [inline it/src/test/scala/com/wavesplatform/it/sync/transactions/IssueTransactionSuite.scala]: Some(null)

**Comment by monroid** [comment]
@ismagin updated

**Pull request merged**